best-practices-badge.md

```diff
@@ -0,0 +1,355 @@
+You are standing in an open field of security options, west of a Badge House, with a boarded front door. There is a small website here. Opening the website reveals a small hackmd.
+
+------
+
+Ensuring the security, sustainability, and maintainability of software projects is critically important. Following industry-standard best practices is a great start to doing that, and the [OpenSSF CII Best Practices badge system][bp] is a reliable way to concretely audit and measure how well the project is doing.
+
+Here's how you can most efficiently secure your project and earn the OpenSSF CII Best Practices badge for your JS project - at a “Passing” level, at a minimum, and “Silver”, ideally. “Gold” is nice, but not practically achievable for most open source projects unless they are lucky or large enough to have multiple maintainers, or a corporation has intensely invested in them. To read more about the tiers, their rationale, and their goals, see [Criteria Discussion][bp-criteria].
+
+First, visit [the login page][bp-login] and log in with your GitHub account.
```

Comment:
"First" seems out of place both because it's part of the summary and it sounds like the first step, but we haven't structured the document in terms of steps. Maybe just edit and have a section called "Logging Into OpenSSF Best Practices"